src/coreComponents/physicsSolvers/python/PySolver.cpp

```diff
@@ -95,10 +95,29 @@ static PyObject * reinit( PySolver * self, PyObject *args )
   Py_RETURN_NONE;
 }
 
+static PyObject * cleanup( PySolver * self, PyObject *args )
+{
+  VERIFY_NON_NULL_SELF( self );
+  VERIFY_INITIALIZED( self );
+  GEOSX_UNUSED_VAR( args );
+
+  double time;
+  if( !PyArg_ParseTuple( args, "d", &time ) )
+  {
+    return nullptr;
+  }
+
+  geosx::DomainPartition & domain = self->group->getGroupByPath< DomainPartition >( "/Problem/domain" );
+  self->group->cleanup( time, 0, 0, 0.0, domain );
+
+  Py_RETURN_NONE;
+}
+
 
 static PyMethodDef PySolver_methods[] = {
   { "execute", (PyCFunction) execute, METH_VARARGS, "solver Step" },
   { "reinit", (PyCFunction) reinit, METH_NOARGS, "re-initialize certain variable depending on the solver being used"},
+  { "cleanup", (PyCFunction) cleanup, METH_VARARGS, "Compute last sismos"},
```

Comment:
Be careful that `PySolver` is the interface for all `Solvers` and not only `WaveSolvers`. a cleanup method must be justified for all `Solvers` then.

Reply:
Also, I think such modification should be part of another PR  since it impacts all `solvers`